src/App.css

```diff
@@ -1,38 +1,81 @@
-.App {
-  text-align: center;
+.bg {
+  background-image: url("https://user-images.githubusercontent.com/41516971/139524483-90a68abb-a68d-4cb3-bf56-5bde965c823c.jpg");
+  width: 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
you can assign all background properties in a single line as well. 
`background: url("imageURL) no-repeat / cover;`